### PR TITLE
NFT Fractionalization on AssetHub Kusama

### PR DESCRIPTION
**Pull request by jsidorenko**

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/3050774

**Review by gilescope — APPROVED** [review]

**Comment by yangwao** [comment]
Yes, looking forward for this!

**Review by joepetrowski — APPROVED** [review]

File: parachains/runtimes/assets/asset-hub-kusama/src/lib.rs [hunk 1]
```diff
@@ -216,7 +216,7 @@ impl pallet_balances::Config for Runtime {
 	type ReserveIdentifier = [u8; 8];
 	type RuntimeHoldReason = RuntimeHoldReason;
 	type FreezeIdentifier = ();
-	type MaxHolds = ConstU32<0>;
+	type MaxHolds = ConstU32<1>;
```
Comment (joepetrowski) [inline parachains/runtimes/assets/asset-hub-kusama/src/lib.rs:219]: Perhaps should have a comment that the hold is from `NftFractionalization`.

Comment (jsidorenko) [inline parachains/runtimes/assets/asset-hub-kusama/src/lib.rs:219]:
```suggestion
	type MaxHolds = ConstU32<1>; // the hold is from `NftFractionalization`
```

Comment (jsidorenko) [reply parachains/runtimes/assets/asset-hub-kusama/src/lib.rs:219]: like this?

**Review by bkchr — APPROVED** [review]

File: parachains/runtimes/assets/asset-hub-kusama/src/lib.rs (hunk 1 above)
Comment (joepetrowski) [reply parachains/runtimes/assets/asset-hub-kusama/src/lib.rs:219]: I would do more like:
```suggestion
	// We allow each account to have holds on it from:
	//   - `NftFractionalization`: 1
	type MaxHolds = ConstU32<1>;
```

**Comment by jsidorenko** [comment]
bot merge

**Pull request merged**